sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/EpidemiologicalDataCasePage.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.sormas.e2etests.pages.application.cases;
+
+import org.openqa.selenium.By;
+
+public class EpidemiologicalDataCasePage {
+  public static final By EXPOSURE_DETAILS_KNOWN =
+      By.cssSelector("#exposureDetailsKnown span:nth-child(1)");
```

Comment:
Please name your selectors related to the items which are mapped.

Take a look to the rest of the classes which are used to mapping to see the pattern used.

We are using:
_COMBOBOX for comboboxes
_DATE for dates
_INPUT for textboxes where data can be inserted
_TEXT or _VALUE or _LABEL for labels where data are stored

Reply:
addressed